qt/widgets/common/src/NotificationService.cpp

```diff
@@ -0,0 +1,68 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "MantidQtWidgets/Common/NotificationService.h"
+#include "MantidKernel/ConfigService.h"
+#include "MantidKernel/Exception.h"
+#include <QApplication>
+
+namespace MantidQt {
+namespace MantidWidgets {
+
+// Key for the "normalize data to bin width" plot option
+const std::string NotificationService::NOTIFICATIONSENABLEDKEY =
```

Comment:
Should this be in all caps snake case? 
```suggestion
const std::string NotificationService::NOTIFICATIONS_ENABLED_KEY =
```